src/ReactiveUI/Routing/RoutingState.cs

```diff
@@ -47,7 +47,7 @@ public RoutingState(IScheduler? scheduler = null)
     /// Gets the current navigation stack, the last element in the
     /// collection being the currently visible ViewModel.
     /// </summary>
-    [IgnoreDataMember]
+    [DataMember]
     public ObservableCollection<IRoutableViewModel> NavigationStack { get; }
```

Comment:
Not sure if just changing `[IgnoreDataMember]` to `[DataMember]` fixes the issue #3322. According to L42, the `NavigationStack` property is only initialized inside the constructor and offers no public setter for a serializer. https://github.com/reactiveui/ReactiveUI/blob/2e3b6000f6cc9a695617dfe2647f87a2f4057044/src/ReactiveUI/Routing/RoutingState.cs#L42